students/uss_tomasz/lesson_05_lists/5_3/bowling_alley.py

```diff
@@ -0,0 +1,26 @@
+from random import randint
+pins_quantity = 10
+balls_quantity = 5
+pins = ['I']*pins_quantity
+
+
+def throw_ball():
+    """Returns randomly positions range. Between them pins will be removed from bowling alley.
+    First value is always smaller than a second value
+    """
+    a, b = randint(1, 10), randint(1, 10)
```

Comment:
**Optional**
Strive for clarity, not for minimal number of lines.
This expression introduces unnecessary binding between these 2 variables, where in fact there is no correlation between them:
```
a, b = randint(1, 10), randint(1, 10)
```
This expression on the other hand clearly shows that they are separate
```
a = randint(1, 10)
b = randint(1, 10)
```